pkgs/by-name/vn/vndr/package.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, fetchpatch
+}:
+
+buildGoModule {
+  pname = "vndr";
+  version = "0.1.2-unstable-2022-12-29";
+
+  src = fetchFromGitHub {
+    owner = "LK4D4";
+    repo = "vndr";
+    rev = "87603e47e8ea2ddac96f508fc9e9d6fc17b198b0";
+    sha256 = "sha256-L7OemAPCv7epOVmjrDDtiGqQqzscm5zj3C6dsZP4uUc=";
+  };
+
```

```suggestion

  postPatch = ''
    go mod init github.com/LK4D4/vndr
  '';
```
Patch isn't really needed, we can just init the module postPatch.